internal/restic/repository.go

```diff
@@ -86,7 +86,7 @@ type MasterIndex interface {
 	// Each returns a channel that yields all blobs known to the index. When
 	// the context is cancelled, the background goroutine terminates. This
 	// blocks any modification of the index.
-	Each(ctx context.Context) <-chan PackedBlob
+	Each(ctx context.Context, fn func(PackedBlob))
```

Comment:
You forgot to update the comment:

    Each runs fn on the all blobs known to the index.

etc.

Reply:
Is changed, and also in `master_index.go`